modules/layers/src/scatterplot-layer/scatterplot-layer.ts

```diff
@@ -183,6 +185,7 @@ export default class ScatterplotLayer<DataT = any, ExtraPropsT extends {} = {}>
 
   state!: {
     model?: Model;
+    uniformStore?: UniformStore<{picking: any}>;
```

Comment:
Put a lot of work into UniformStore support for uniform types, hurts to see `any` here :)

Reply:
The uniformStore is designed to be independent of the device so  that you can initialize it here instead of in the constructor. Saves a line of code and lots of duplicate typing.